docs/source/manifests.md

```diff
@@ -557,6 +557,14 @@ The content of the AWS SecretsManager secret must be a JSON containing two value
 },
 ```
 
+The archive secret would then need to be referenced in the deployment manifest:
+
+```yaml
+...
+archiveSecret: example-archive-credentials-modules
```

Comment:
can you link to the definition of the archiveSecret elsewhere in the docs?

Reply:
Which part do you want to link to? I added a link from the deployment manifest to the Archive Secret section